patches/tock/01-persistent-storage.patch

```diff
@@ -1,8 +1,8 @@
-diff --git a/boards/nordic/nrf52840_dongle/src/main.rs b/boards/nordic/nrf52840_dongle/src/main.rs
```

Comment:
This patch could be heavily simplified because only the last chunk needs to be applied on the stock kernel.
It could be pre-applied on the `boards/` directory with this PR.
But let's keep it for a future PR.

Reply:
Got it.